### PR TITLE
Feat/highlight

### PR DESCRIPTION
**Pull request by ArthurEncr**
# Description

- Style for highlighted favorite card
- Logic to highlight and dehighlight a favorite card
- Global changes to overflow menu
- Changes in the favorites cards mock and card type 
JIRA ticket: https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-323

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/components/FavoriteActivities/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.main-content {
+    background-color: #F3F3F5 !important;
+    padding: 32px 0;
+}
```
Comment (mgaseta) [inline src/components/FavoriteActivities/styles.css:4]: I think this has been a conflict resolution after this branch was updated, please revert it back.

File: src/components/RecentActivities/styles.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.recent-activity {
+    padding: 32px 0 !important;
+    background-color: #ffffff !important;
+}
+
+.activity-table {
+    padding: 0px !important;
+}
```
Comment (mgaseta) [inline src/components/RecentActivities/styles.css:8]: I think this has been a conflict resolution after this branch was updated, please revert it back.

File: src/layout/PublicLayout/styles.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.mainContainer {
+  padding-top: 0px;
+}
```
Comment (mgaseta) [inline src/layout/PublicLayout/styles.css:3]: I think this has been a conflict resolution after this branch was updated, please revert it back.

File: src/theme/components-overrides.scss [hunk 4]
```diff
@@ -223,3 +223,23 @@ $types: 'error', 'info', 'success', 'warning';
     border-bottom-color: var(--bcgov-notification-border-#{$notificationType});
     border-right-color: var(--bcgov-notification-border-#{$notificationType})}
 }
+
+.card-overflow{
+  border-radius: 4px;
+}
```
Comment (mgaseta) [inline src/theme/components-overrides.scss:229]: If this rule is specific only to the card component, it should be moved to the component style file.

File: src/types/Card.ts [hunk 5]
```diff
@@ -1,7 +1,10 @@
...
-  icon: string;
-  header: string;
-  description: string;
+type CardType = {
+    id: string;
+    icon: string;
+    header: string;
+    description: string;
+    link: string;
+    highlighted?: boolean;
```
Comment (mgaseta) [inline src/types/Card.ts:7]: Adjust tabs here.

**Review by mgaseta — CHANGES_REQUESTED** [review]
Great job 🎉! Add a fez markups mainly related to code style and organization.

**Review by RMCampos — APPROVED** [review]
Great job!! Very good 🥳

File: src/App.tsx [hunk 6]
```diff
@@ -35,7 +35,7 @@ const App: React.FC = () => {
           <Route
             path="/dashboard"
             element={(
-              <ProtectedRoute signed={signed}>
+              <ProtectedRoute signed>
```
Comment (mgaseta) [inline src/App.tsx:38]: Revert this change.

**Review by mgaseta — CHANGES_REQUESTED** [review]

**Review by mgaseta — APPROVED** [review]

**Pull request merged**